Core/clim-core/commands/tables.lisp

```diff
@@ -149,10 +149,8 @@
           (map-func command-table)))))
 
 (defun command-present-in-command-table-p (command-name command-table)
-  (let ((table (find-command-table command-table)))
-    (if (gethash command-name (slot-value table 'commands))
-        table
-        nil)))
+  (let ((ht (slot-value (find-command-table command-table) 'commands)))
+    (gethash command-name ht nil)))
```

Comment:
This should probably not return two values.

Reply:
fixed (using nth-value 1 (gethash))